apps/ftu/index.html

```diff
@@ -441,28 +434,6 @@ <h2 id="time-zone-title">
             </li>
           </ul>
         </article>
-        <gaia-subheader skin="organic">
-          <span data-l10n-id="otherAccount">Other accounts</span>
-        </gaia-subheader>
-        <article data-type="list">
-          <ul id="fb_container">
-            <li id="fb_import" class="importOption">
-              <button id="fb-import-button" class="icon icon-fb" data-l10n-id="importFb2">
-                  Facebook
-              </button>
-              <p id="no-network" class="hidden" data-l10n-id="noConnection11">
-                To import Facebook friends connect to Wi-Fi or data network
-              </p>
-            </li>
-            <li id="fb_after_import1" class="hidden">
-              <p data-l10n-id="importFb2">Facebook</p>
-              <p id="fb_import_feedback"></p>
-            </li>
-            <li class="importOption hidden" id="fb_after_import2">
-              <p data-l10n-id="afterFbImport"></p>
-            </li>
-          </ul>
-        </article>
```

Comment:
Do we need [1] and all panels related with FB? Is `ServicesLauncher` needed anymore?

[1] https://github.com/fcampo/gaia/blob/fb_out_ftu_1175491/apps/ftu/index.html#L751